docs/conf/portland/2018/team.rst

```diff
@@ -25,67 +45,50 @@ Mikey Ariel
 
 .. include:: /include/bios/mikey.rst
 
-Samuel Wright
-~~~~~~~~~~~~~
+Mo Nishiyama
+~~~~~~~~~~~~
 
-.. include:: /include/bios/sam.rst
+.. include:: /include/bios/mo.rst
 
-Becky Yoose
-~~~~~~~~~~~
+Nikolas Blanchet
+~~~~~~~~~~~~~~~~
 
-.. include:: /include/bios/becky.rst
 
 Rose Williams
 ~~~~~~~~~~~~~
 
 .. include:: /include/bios/rose.rst
 
-Christy Lutz
-~~~~~~~~~~~~~
-
-.. include:: /include/bios/christy.rst
-
-Mo Nishiyama
+Samuel Wright
 ~~~~~~~~~~~~~
 
-.. include:: /include/bios/mo.rst
+.. include:: /include/bios/sam.rst
 
 Shaun McCance
 ~~~~~~~~~~~~~
 
 .. include:: /include/bios/shaun.rst
 
-Alicia Duncan
-~~~~~~~~~~~~~
-
-.. include:: /include/bios/alicia.rst
 
 Primary Roles
 -------------
 
-You can read descriptions of all the roles in our :doc:`/organizer-guide/confs/event-roles` doc.
-
-Filled Roles
-~~~~~~~~~~~~~
-
 * **Conference chair** - Eric Holscher
-* **Swag coordinator** - Mikey Ariel
-* **Speaker wrangler** - Kelly O'Brien
+* **Code of conduct response** - See `code of conduct <http://www.writethedocs.org/code-of-conduct/#staff-contact-information>`_ page
+* **Communication chair** - Mikey Ariel
+* **Emcee** - Nikolas Blanchet
+* **Explore Portland coordinators** - Mo Nishiyama and Alicia Duncan
+* **Hike coordinator** - Eric Holscher
 * **Infrastructure chair** - Samuel Wright
-* **Writing Day coordinator** - Shaun McCance
+* **Job Fair coordinator** - Eric Holscher
 * **Lightning Talk coordinator** - Rose Williams
-* **Communication chair** - Mikey Ariel
-* **Volunteer coordinator** - Becky Yoose
+* **Social media chair** - Mikey Ariel
+* **Speaker wrangler** - Kelly O'Brien
```

Comment:
We decided to stop using "wrangler", I guess I missed this in the search/replace - needs to be "coordinator" everywhere